data/systemd/Makefile

```diff
@@ -13,3 +14,13 @@ all:
 		sed s:@SNAP_MOUNTDIR@:${SNAP_MOUNTDIR}:g > snapd.refresh.service
 	cat snapd.autoimport.service.in | \
 		sed s:@bindir@:${BINDIR}:g > snapd.autoimport.service
+
+install:
+	install -d ${DESTDIR}/${SYSTEMDSYSTEMUNITDIR}
+	install --mode=0644 snapd.refresh.timer ${DESTDIR}/${SYSTEMDSYSTEMUNITDIR}
+	install --mode=0644 snapd.refresh.service ${DESTDIR}/${SYSTEMDSYSTEMUNITDIR}
+	install --mode=0644 snapd.autoimport.service ${DESTDIR}/${SYSTEMDSYSTEMUNITDIR}
+	install --mode=0644 *.socket ${DESTDIR}/${SYSTEMDSYSTEMUNITDIR}
+	install --mode=0644 snapd.service ${DESTDIR}/${SYSTEMDSYSTEMUNITDIR}
+	install --mode=0644 snapd.system-shutdown.service ${DESTDIR}/${SYSTEMDSYSTEMUNITDIR}
```

Comment:
This is new to me. This should be templated too, even if it doesn't currently apply to non-snappy systems. You never know who is going to make a snap-based system. :)